src/platform/i386/kernel.c

```diff
@@ -13,6 +13,10 @@
 #include <component.h>
 #include <thd.h>
 
+#define ADDR_STR_LEN	8
+#define CMDLINE_MAX_LEN	32
+#define CMDLINE_REQ_LEN	(ADDR_STR_LEN * 2 + 1)
```

Comment:
Is it only the line 16 that's strange or all 3 lines? I've mainly used TAB in Vim in all cases I guess. But I missed to align line 16 correctly.